src/index.html

```diff
@@ -1,12 +1,492 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img class="logo__img"
+             src="/images/logo.png"
+             alt="logo"
+        >
```

Comment:
```suggestion
        <img
          class="logo__img"
          src="/images/logo.png"
          alt="logo"
        >
```
Дуже багато пробілів тут